zipkin-query-service/src/main/scala/com/twitter/zipkin/builder/QueryServiceBuilder.scala

```diff
@@ -41,9 +41,7 @@ case class QueryServiceBuilder(
     }
 
     object UseOnceFactory extends com.twitter.app.App with ZipkinQueryServerFactory
-    UseOnceFactory.nonExitingMain(Array(
-      "zipkin.queryService.port", serverBuilder.serverAddress + ":" + serverBuilder.serverPort
```

Comment:
missing `.getHostAddress`, but also awkard way to set the value.. below is less awkward